09Ianuarie/src/ro/sci/bv/InputDataValidation/InputDataValidation.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * This program asks the user to enter the current time
+ * The user must enter two numbers - one for hours and one for minutes
+ * The program will validate the entry data and will output it if it's correct
+ *
+ * @author Bogdan Butuza
+ */
+package ro.sci.bv.InputDataValidation;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.util.Scanner;
+public class InputDataValidation {
+
+    /**
+     * Method checkData verifies if both values are valid ones
+     * If correct, it puts hour and minutes in the right format and prints it to the user
+     * @param hour a short variable, representing the given hour
+     * @param minutes a short variable, representing the given minutes
+     */
+    private static void checkData(short hour, short minutes) {
+        if  ((hour>=0 && hour<=23) && (minutes>=0 && minutes<=59)) {
```

Comment:
again, this method does too much. It's named "checkData" but it also formats and displays the value.
You should have a method checkData that only verifies if hour and minutes have valid values and then have another method that convertsAndPrints them or maybe even doing that in the main function.